### PR TITLE
Updating version in installing nmstate  CLI output

### PR DESCRIPTION
**Pull request by skrthomas**
Creating this PR based off QE observation that the version number in the CLI output needs updating in 4.12. It also needs in 4.11.

Version(s):

4.11 only

Issue:

QE feedback about version in 4.12 PR: https://github.com/openshift/openshift-docs/pull/51941#issuecomment-1286383453
Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 1]
```diff
@@ -99,5 +99,5 @@ oc get clusterserviceversion -n openshift-nmstate \
 [source, terminal]
 ----
 Name                                             Phase
-kubernetes-nmstate-operator.4.10.0-202203120157   Succeeded
+kubernetes-nmstate-operator.4.10.0-202208120157   Succeeded
```
Comment (qiowang721) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:102]: maybe you mean kubernetes-nmstate-operator.4.11.0-202208120157? ^ ^

**Comment by kelbrown20** [comment]
Looks great! 
/lgtm

**Pull request merged**